electricitymap/contrib/lib/models/event_lists.py

```diff
@@ -204,18 +204,21 @@ def append(
     def merge_production_breakdowns(
         ungrouped_production_breakdowns: List["ProductionBreakdownList"],
         logger: Logger,
+        matching_timestamps_only: bool = False,
     ) -> "ProductionBreakdownList":
         """
         Given multiple parser outputs, sum the production and storage
         of corresponding datetimes to create a unique production breakdown list.
         Sources will be aggregated in a comma-separated string. Ex: "entsoe, eia".
         There should be only one zone in the list of production breakdowns.
+        Matching timestamps only will only keep the timestamps where all the production breakdowns have data.
```

Comment:
Nice, this means we might be able to move forward with splitting up AR after this.

Reply:
Yes I think this would simplify things quite a lot!